src/index.js

```diff
@@ -46,6 +46,11 @@ app.use('/api/v1/users', userRouter);
 app.use('/api/v1/trips', tripsRouter);
 app.use('/api/v1/facilities', facilitiesRouter);
 
+
+app.use((req, res) => res.status(404).send({ status: 404, error: `Route ${req.url} Not found.` }));
```

Comment:
hey @Baraka-Mugisha , could you add the internalization to this response

Reply:
It is done